Reviewing some texts and adding error treatment step (#56)
indigotech/taq-github-bot: src/data/local/github/tracks-node/track-3/steps/step-1.md

Let's start with baby steps, by creating the mutation prototype, with no integration with the dabatase (yet).

> Let's call the mutation `login`. It should receive an e-mail and a password as parameters and return the following structure on body:

Reviewer: ```suggestion
Let's call the mutation `login`. It should receive an e-mail and a password as parameters and, in case of success, 
 should return the following structure:
```

**Note:** Did you notice that there is a `data` object wrapping the `Login` response? Don't worry, it's the GraphQL response format. It wraps all the successful response data inside the `data` object and all the errors in an `errors` array of objects. You can read more about it [here](https://github.com/graphql/graphql-spec/blob/master/spec/Section%207%20--%20Response.md#data).

> For now, you can return the above structure diretcly on your code populated with some mock data.

Reviewer: ```suggestion
For now, you can return the above structure directly on your code populated with some mock data.
```
